### PR TITLE
Fixed readd last action.

### PR DESCRIPTION
**Pull request by normanjaeckel**

### CODE REVIEW

File: openslides_backend/action/actions/list_of_speakers/re_add_last.py [hunk 1]
```diff
@@ -1,63 +1,83 @@
...
         title="Re-add last speaker",
-        description="Adds the last speaker as new speaker.",
+        description="Moves the last speaker back to the top of the list.",
     )
 
-    def base_update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        # Check if list of speakers has speakers.
+        list_of_speakers_id = instance["id"]
         list_of_speakers = self.fetch_model(
```
Comment (jsangmeister) [inline openslides_backend/action/actions/list_of_speakers/re_add_last.py:28]: This fetch is unnecessary, you can just check if the filter call returned a result or not.

File: openslides_backend/action/actions/list_of_speakers/re_add_last.py [hunk 2]
```diff
@@ -1,63 +1,83 @@
...
+                f"List of speakers {list_of_speakers_id} has no speakers."
+            )
+
+        # Fetch last speaker.
         last_speakers = sorted(
             self.datastore.filter(
-                Collection("speaker"),
-                filter_obj,
+                self.model.collection,
+                FilterOperator("end_time", "!=", None),
```
Comment (jsangmeister) [inline openslides_backend/action/actions/list_of_speakers/re_add_last.py:41]: Filter for `list_of_speakers_id` is missing

File: openslides_backend/action/actions/list_of_speakers/re_add_last.py [hunk 3]
```diff
@@ -1,63 +1,84 @@
...
+            ),
+            lock_result=True,
+        )["exists"]:
+            raise ActionException(
+                f"User {last_speaker['user_id']} is already on the list of speakers."
+            )
+
+        # Fetch all speakers and sort them.
+        all_speakers = sorted(
+            self.datastore.filter(
```
Comment (jsangmeister) [inline openslides_backend/action/actions/list_of_speakers/re_add_last.py:68]: Just use the `min` function here

**Review by jsangmeister — CHANGES_REQUESTED** [review]
Alternative to prevent the multitude of datastore calls: Just fetch all speakers from this LOS and evaluate everything (whether a last speaker exists, whether he is already on the list again, what the min weight is) locally. Would probably be the cleanest solution.

**Comment by normanjaeckel** [comment]
All done.

**Review by jsangmeister — APPROVED** [review]

**Pull request merged**